cvrf2csaf/config/config.yaml

```
@@ -9,7 +9,7 @@ csaf_version: "2.0"
 
 # Document Publisher
 publisher_name: Publisher Name
-publisher_namespace: Publisher Namespace
+publisher_namespace: http://publisher.name.space
```

space seems to be a valid gTLD - so we SHOULD NOT use that
```suggestion
publisher_namespace: https://example.com